### PR TITLE
Report Day by Day

### PR DESCRIPTION
**Pull request by abdullahallmehedi**

### CODE REVIEW

**Comment by rofiuddin** [comment]
okay

File: app/assets/javascripts/supervisor/daily_reports.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (chuanhtuan) [inline app/assets/javascripts/supervisor/daily_reports.coffee]: remove unused file

File: app/assets/stylesheets/supervisor/daily_reports.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the supervisor/daily_reports controller here.
```
Comment (chuanhtuan) [inline app/assets/stylesheets/supervisor/daily_reports.scss]: remove unused file

File: app/models/daily_report.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+class DailyReport < ActiveRecord::Base
```
Comment (chuanhtuan) [inline app/models/daily_report.rb]: I think, It's wrong
This is UserTask model

Comment (abdullahallmehedi) [inline app/models/daily_report.rb]: We had a feature "Report day by day". Will report be a model??

**Pull request closed without merging**